### ISSUE TITLE
[beta] Can't affect a particular key binding

### ISSUE THREAD

**Issue by kud** [wulkano/kap#733]
**macOS version:** `macOS 10.14.6`
**Kap version:** `3.0.0-beta.6`

#### Steps to reproduce
Preferences > Trigger Kap: `⌘+⇧+(`

#### Current behaviour
On an azerty keyboard, it doesn't work. Note: ⇧+( does 5

#### Expected behaviour
Working

**Comment by topalavlad** [comment]
If I understand correctly, the problem is that ⌘⇧5 cannot be used as a trigger, but I think that's expected since it's a [os x keystroke](https://support.apple.com/en-us/HT201361). Same goes for ⌘⇧4. @kud?

**Comment by karaggeorge** [comment]
Yeah, I've had the same issue since Mojave, but like @topalavlad said, it's mapped to a system keystroke, so I don't know if there's a workaround for that
